nvflare/private/fed/server/server_app_runner.py

```diff
@@ -42,7 +43,11 @@ def _set_up_run_config(workspace: Workspace, server, conf):
     server.handlers = conf.handlers
 
 
-class ServerAppRunner:
+class ServerAppRunner(Runner):
+    def __init__(self, server) -> None:
+        super().__init__()
+        self.server = server
```

Comment:
If we have this in init method, maybe other method of this class don't need to take "server" as input argument anymore?

Reply:
changed.